### ISSUE TITLE
`latest` image on Docker Hub is extremely out-of-date

### ISSUE THREAD

**Issue by magneticflux-** [CachetHQ/Docker#313]
After trying to figure out why my metrics chart wasn't showing and finding that it wasn't using the API properly, I discovered that I was not running the "latest" version that I thought I was. Rather, I was running a 2-year-old version build from the `master` branch (which has been unused). It seems that `2.4` is the default branch now, and the build files should be changed accordingly.

**Comment by joca-bt** [comment]
++

**Comment by joca-bt** [comment]
Yes, but then we are stuck on a version that doesn't have incident updates.
